clang/test/3C/checkedregionvoid.c

```diff
@@ -1,4 +1,7 @@
-// RUN: 3c -addcr  %s -- | FileCheck -match-full-lines --check-prefixes="CHECK" %s
+// RUN: 3c -addcr %s | FileCheck -match-full-lines --check-prefixes="CHECK" %s
+
+// Currently not running clang on the output,
+// since it cannot find "dummy.h", even with -L%S
```

Comment:
There is no `dummy.h` in the test directory? If that's right, I think we should delete its usage.

Reply:
There is a `dummy.h` in the test directory, required for the test as it is. I couldn't get clang to recognize it from the testing harness.

Reply:
If it's a problem then we should just get rid of it. I don't see the benefit of having it when you can just inline its contents in the test.

Reply:
I removed it